obfuscation.go

```diff
@@ -18,7 +18,7 @@ type OnionErrorEncrypter struct {
 func NewOnionErrorEncrypter(router *Router,
 	ephemeralKey *btcec.PublicKey) (*OnionErrorEncrypter, error) {
 
-	sharedSecret, err := router.generateSharedSecret(ephemeralKey)
+	sharedSecret, err := router.generateSharedSecret(ephemeralKey, nil)
```

Comment:
🙏 can we also asseblief update `NewOnionEncrypter` to take a blinding key param?

Reply:
natuurlik 👍

Reply:
done!